imagetest/cmd/startup.ps1

```diff
@@ -0,0 +1,52 @@
+# This is temporary until the next windows build happens which will contain the new metadata-scripts package
```

Comment:
let's just coordinate them then and not check in temp scripts. you can run your development testing against images which do have the necessary changes